src/main/java/io/cdap/plugin/dlp/configs/DlpTransformConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.cdap.plugin.dlp.configs;
+
+import com.google.privacy.dlp.v2.PrimitiveTransformation;
+import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.cdap.etl.api.FailureCollector;
+
+import java.util.List;
+
+/**
+ *
+ */
+public interface DlpTransformConfig {
+
+  PrimitiveTransformation toPrimitiveTransform();
```

Comment:
each method in an interface should have javadoc

Reply:
done